libs/components/lookup/src/lib/modules/autocomplete/autocomplete-input.directive.ts

```diff
@@ -166,66 +159,44 @@ export class SkyAutocompleteInputDirective
 
   #_value: any;
 
-  constructor(elementRef: ElementRef, renderer: Renderer2) {
-    this.#elementRef = elementRef;
-    this.#renderer = renderer;
-
-    this.#blur = new Subject<void>();
-    this.#focus = new Subject<void>();
-    this.#textChanges = new Subject<SkyAutocompleteInputTextChange>();
-
-    this.#blurObs = this.#blur.asObservable();
-    this.#focusObs = this.#focus.asObservable();
-    this.#textChangesObs = this.#textChanges.asObservable();
-  }
-
-  public ngOnInit() {
-    const element = this.#elementRef.nativeElement;
-
+  public ngOnInit(): void {
     this.#setAttributes(this.#elementRef);
-
-    observableFromEvent(element, 'input')
-      .pipe(takeUntil(this.#ngUnsubscribe))
-      .subscribe(() => {
-        if (!this.disabled) {
-          this.#textChanges.next({
-            value: this.#elementRef.nativeElement.value,
-          });
-        }
-      });
-
-    observableFromEvent(element, 'blur')
-      .pipe(takeUntil(this.#ngUnsubscribe))
-      .subscribe(() => {
-        if (!this.disabled) {
-          this.#blur.next();
-        }
-      });
-
-    observableFromEvent(element, 'focus')
-      .pipe(takeUntil(this.#ngUnsubscribe))
-      .subscribe(() => {
-        if (!this.disabled) {
-          this.#focus.next();
-        }
-      });
-
-    observableFromEvent(element, 'change')
-      .pipe(takeUntil(this.#ngUnsubscribe))
-      .subscribe(() => {
-        /* istanbul ignore else */
-        if (!this.disabled) {
-          this.#isFirstChange = false;
-        }
-      });
   }
 
   public ngOnDestroy(): void {
     this.#blur.complete();
     this.#textChanges.complete();
+  }
 
-    this.#ngUnsubscribe.next();
-    this.#ngUnsubscribe.complete();
+  @HostListener('blur')
+  public onBlurHandler(): void {
```

Comment:
Is there an accessibility reason we switched from `observableFromEvent` to `HostListener`? If so, these can probably be `protected` instead of `public`.

Reply:
I made these changes when I wasn't seeing the result I expected, it ended up being a different issue, and I've reverted these changes.